langgraph/demo.py

```
@@ -0,0 +1,151 @@
+    return f"These are the members from team {team_name}:{', '.join(team_members)}."
+
+def list_teams():
+    """Returns all available teams in exact format."""
+    teams_query_result = directory_db_query("SELECT name FROM teams")
+    teams = [item[0] for item in teams_query_result]
+
+    return f"The list of all teams in the company are {', '.join(teams)}"
+
+def show_team(team_name: str):
+    """Gives a description for only one specific team. For multiple teams, need to call this tool repeatedly"""
+    team_query_result = directory_db_query(f"SELECT name, team_function FROM teams WHERE name = '{team_name}' LIMIT 1")
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)